packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1186,19 +1195,25 @@ class _TabBarState extends State<TabBar> {
       color = Colors.white;
     }
 
+    final bool primaryWithLabelIndicator = widget._isPrimary && indicatorSize == TabBarIndicatorSize.label;
+    final double effectiveIndicatorWeight = theme.useMaterial3
```

Comment:
This looks like if `useMaterial3` is true, the user provided value of widget.indicatorWeight is just ignored completely?

Reply:
Good catch!